docker-compose.yml

```diff
@@ -16,8 +16,8 @@ services:
       - chain-data:/data
     command: --dev --alice --validator --unsafe-ws-external --unsafe-rpc-external --rpc-methods Unsafe --rpc-cors=all --log runtime --base-path /data
     ports:
-      - '127.0.0.1:9944:9944'
-      - '127.0.0.1:9933:9933'
+      - 9944:9944
+      - 9933:9933
```

Comment:
It may be a non-issue, but since we're using `--rpc-methods Unsafe` and `--rpc-cors=all` flags, perhaps we should add a comment here somewhere that this is not a safe production setup.

Reply:
Perhaps joystream-node container doesn't in-fact need to listen on `0.0.0.0`

Reply:
But more importantly why do we need to enable the unsafe methods?

Reply:
I remember I suggested that because it allows calling `author_rotateKeys` from localhost / Pioneer when the node is running as docker container. It also allows testing [status server](https://github.com/Joystream/status-endpoint-joystream/tree/development), as it uses `api.rpc.system.peers`

Reply:
Ah yes thanks for reminder. I think its best to have the safer default of not allowing the "unsafe rpc" and when the unsafe rpc commands are needed the test node can be started with `./tests/network-tests/run-test-node-docker.sh` which allows the Unsafe commands.

Done in ff8f82be895d18b4529ee4ac73795713da3066a5